site/src/hooks/useFinalizedHeight.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { useChainApi } from "../utils/hooks/chain/useChainApi";
+
+export default function useFinalizedHeight() {
+  const api = useChainApi();
+  const [finalizedHeight, setFinalizedHeight] = useState(null);
+
+  useEffect(() => {
+    if (!api) {
+      return;
+    }
+
+    let unsub = null;
+    api.rpc.chain
+      .subscribeFinalizedHeads(async (header) => {
+        setFinalizedHeight(header.number.toNumber());
+      })
+      .then((result) => (unsub = result));
+
+    return () => {
+      if (unsub) {
+        unsub();
+      }
+    };
+  }, [api]);
+
+  return finalizedHeight;
```

Comment:
I think we can store `finalizedHeight` to redux, and we can rename this hook to `useSubFinalizedHeight`. Then we should use `useSelector` when we need the finalized height. Call this hook when we start the whole app.